src/hooks/useEtherscan.js

```diff
@@ -0,0 +1,35 @@
+import { isString } from 'lodash';
+import { useCallback, useMemo } from 'react';
+import { Linking } from 'react-native';
+import useAccountSettings from './useAccountSettings';
+import { ethereumUtils } from '@rainbow-me/utils';
+
+export default function useEtherscan() {
+  const { network } = useAccountSettings();
+  const etherscanHost = useMemo(
```

Comment:
Assuming it's just a string and it's cheap, I would suggest not using memorization here.

Therefore, I cannot see then the reason why `openTokenEtherscanURL` and `openTransactionEtherscanURL` won't be just static functions that can be provided as some helper?

Reply:
Agreed, let's make this a util!

Reply:
@jinchung @osdnk okie! made the change here -> https://github.com/rainbow-me/rainbow/pull/1552/commits/69ed637d61b01dd345d380799591443049963125

lemme know if u have further tweaks youd like me to make